.circleci/config.yml

```diff
@@ -9,12 +9,12 @@ version: 2.1
 parameters:
   ubuntu-2004-docker-image:
     type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2004-24
-    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:b8b645fa7ab40d55f2d16eac295d16ca01ec51d32be7d668ae6eaecd47dbd763"
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2004-25
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:b3f321fb2d8e7a41ca9328672061c1840e5cd3fb5be503aa158d1c508deacf0a"
   ubuntu-2204-docker-image:
     type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204-9
-    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:80247de9655b1f39afd4ac22b14266bc9b9a0d64b283ae8fb9cb5b8250e4e77d"
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204-10
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:ba2d878c26d681a7d6c1922258b47c2e5dd61d0e46ab4c6a4862b473e61997b6"
   ubuntu-2204-clang-docker-image:
     type: string
     # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-8
```

Comment:
Odd, were these changed reverted @rodiazet? I thought you had bumped these previously?
@cameel we dynamically link against evmone, and the only jobs that use them (soltest and friends) have been switched to 2404 images, so that's likely the reason it passes. I.e. it's built on 2204, but executed on 2404 where all the prerequisites are met.

Reply:
It looks that all this re-targeting and re-basing I made caused the problem. It was removed by https://github.com/ethereum/solidity/compare/928690807a919d487a5d0a8aec2703802e7be8fc..1006992fbcf7443ad991026c5d7c536273957b60
Just added it back.